src/framework/shortcuts/view/editshortcutmodel.cpp

```diff
@@ -110,6 +111,31 @@ void EditShortcutModel::inputKey(Qt::Key key, Qt::KeyboardModifiers modifiers)
     emit newSequenceChanged();
 }
 
+bool EditShortcutModel::checkNotSpecialKey(Qt::Key key)
+{
+    // Few special keys for which the Shift modifier should not be removed
+    switch (key) {
+    case Qt::Key_Up:
+    case Qt::Key_Down:
+    case Qt::Key_Left:
+    case Qt::Key_Right:
+    case Qt::Key_Insert:
+    case Qt::Key_Delete:
+    case Qt::Key_Home:
+    case Qt::Key_End:
+    case Qt::Key_PageUp:
+    case Qt::Key_PageDown:
+    case Qt::Key_Space:
+    case Qt::Key_Escape:
+    case Qt::Key_F1: case Qt::Key_F2:  case Qt::Key_F3:  case Qt::Key_F4:
+    case Qt::Key_F5: case Qt::Key_F6:  case Qt::Key_F7:  case Qt::Key_F8:
+    case Qt::Key_F9: case Qt::Key_F10: case Qt::Key_F11: case Qt::Key_F12:
```

Comment:
I think it's better to just list these vertically like the rest. It means a bit more vertical scrolling, but that's easier than transitioning from reading down to reading across.

Reply:
It is a pity that `case Qt::Key_F1 ... case Qt::Key_F12:` isn't working on non-GNU compilers

Reply:
This would work:
```C++
if (Qt::Key_F1 <= key && key <= Qt::Key_F12)
```

But I think it's better to list the cases separately rather than rely on details of how Qt's key enum is implemented.

Reply:
Qt also has F13 … F35. Perhaps we should list those too, just for consistency. FWIW, old-ish Mac keyboards go up to F19 iirc.